product-list-BE/routes/main.js

```diff
@@ -0,0 +1,142 @@
+const express = require('express');
+const router = express.Router();
+const Product = require('../models/products');
+const Review = require('../models/reviews');
```

Comment:
convention says we should name our schema files in singular. Your schema represents one product and one review. Not many. Also, makes more sense. 
```suggestion
const Product = require('../models/product');
const Review = require('../models/review');
```